CHANGELOG.md

```diff
@@ -1,6 +1,99 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.11.0 (2019-03-13)
+
+This release comes with experimental garbage collection and Git commit signing:
+
+1. Experimental garbage collection of cluster resources. When providing the
+   `--sync-garbage-collection` flag, namespace-scoped cluster resources no
```

Comment:
I didn't notice this bit -- it is all resources, not just those in namespaces (unless you know something I don't? :-)

Reply:
I recall reading a test which verified this. I'm probably wrong, I will fix this tomorrow.